### PR TITLE
PennyPincher ~ Microverse Ruby On Rails Capstone Project

### PR DESCRIPTION
**Pull request by PrantoshB**
# Hello There 👋🏽 

hello

In this PR, I have been able to :
- Add splash page
- Add categories pages
- Add categories model
- Add categorie#index route and view
- Add category #new route and view
- Add payment pages
- Add payment#index route and view
- Add payment#new route and view
- Add authentication with `devise`
- Add authorization with `CanCanCan`
- Add unit and integration test for main components

### Prantosh

- ## [Live Demo 🚀](https://penny-pincher-tfv3.onrender.com/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,143 @@
...
+If you like this project, please leave a start to support
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this great learning experience !
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Omar-Muhamad) [inline README.md:138]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=on%20Behance.-,The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.,-Interactions), Kindly give credits to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). you can add it here in the `Acknowledgments` section.

File: README.md [hunk 2]
```diff
@@ -1 +1,143 @@
...
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
```
Comment (Omar-Muhamad) [inline README.md:49]: - [ ] Please also add the deployment link that you provided in the pull request summary here in the README file under `Live Demo 🚀` section.

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @PrantoshB,

### Nice job, you are almost there.

giphy (2)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Eventhough you did an outstanding work in this app there is some aspects to improve.
As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=the%20project%20features.-,you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.,-For%20recording%20the), For the video presentaion you provided, Please mension a piece of the code you finds interesting or is proud of making.

image
  
- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- [ ] N/A.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (PrantoshB) [reply README.md]: # Done ✅

File: README.md (hunk 2 above)
Comment (PrantoshB) [reply README.md]: # Done ✅

**Review by gemmen29 — APPROVED** [review]
Hi @PrantoshB, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**